psl-core/src/main/java/org/linqs/psl/model/rule/arithmetic/UnweightedArithmeticRule.java

```diff
@@ -63,6 +63,7 @@ protected UnweightedGroundArithmeticRule makeGroundRule(List<Float> coefficients
 
     @Override
     public WeightedRule relax(float weight, boolean squared) {
+        unregister();
```

Comment:
Add in a javadoc method comment in the `UnweightedRule`interface that mentions that relax() should unregister the context rule.